pkg/service/regression/regression.go

```diff
@@ -384,12 +405,10 @@ func (r *Regression) testGrpc(ctx context.Context, cid, runId, id, app string, r
 	}
 
 	if !pass {
-		logger := pp.New()
```

Comment:
pp logging library is used to log pretty logs. Can you use this library to log the test-result in the log file